.mailmap

```diff
@@ -20,16 +20,19 @@ Darshan Chaudhary <deathbullet@gmail.com>
 Devarshi Sathiya <devarshisathiya5@gmail.com>
 Florian Koch <f0@users.noreply.github.com>
 George Kontridze <gkontridze@plaid.com>
+Hui Kong <konghui@live.cn>
```

Comment:
This should have been caught in the `.mailmap` file? I remember fixing this for the v1.8.x release recently.

Reply:
This is the mailmap file :-) 

Looks like I've handled it differently here from how you did for v1.8 though, I can fix that up. Good catch.